packages/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -601,13 +625,13 @@ export class VerticalBarChartBase extends React.Component<IVerticalBarChartProps
   }
 
   private _createStringBars(containerHeight: number, containerWidth: number, xElement: SVGElement): JSX.Element[] {
```

Comment:
>_createStringBars [](http://example.com/codeflow?start=10&length=17)

Can we reduce code duplication in these 3 functions creating bar for numeric/string/date.

Reply:
This i will take up in separate PR.